src/DrevOps/Sniffs/TestingPractices/DataProviderOrderSniff.php

```
@@ -0,0 +1,421 @@
+          if ($token['line'] === $provider_line && $token['code'] === T_STRING) {
+            $error_ptr = (int) $ptr;
+            break;
+          }
+        }
+
+        if ($error_ptr !== NULL) {
+          $phpcsFile->addError($error_message, $error_ptr, $error_code, $data);
+        }
+      }
+    }
+  }
```

PHPMD (2.15.0)

The method validateOrder() has a Cyclomatic Complexity of 14. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)